tests/test_ec2_manager.py

```diff
@@ -256,6 +256,17 @@ def test_list_detail_subnets(em):
     assert len(subnets) > 0
 
 
+def test_get_subnet_ids_with_vpc_id(em):
+    # Given
+    default_vpc_id = em.get_default_vpc()['VpcId']
+
+    # When
+    subnets = em.get_subnet_ids_with_vpc_id(default_vpc_id)
+
+    # Then
+    assert len(subnets) > 0
+    pprint(subnets)
```

Comment:
it would be better to use `print(json.dumps(subents))` since it's able to inject online json parser directly.

Reply:
thanks for suggestion. will use json dumps!